### ISSUE TITLE
Host Fontawesome locally

### ISSUE THREAD

**Issue by TadeSF** [DeutscheModelUnitedNations/munify#80]
More flexibility, less dependencie issues with password protected npm packages, long-term-solution if FA-Subscription is not renewed.

Prerequisite: Gitcrypt or similar enables to store licenced FA Pro Icons in Repository without having licence issues in open repository (see #79)

**Comment by TadeSF** [comment]
Duplicate of #63
